.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Envorioment (please complete the following information):**
+ - bnd-Version: [e.g. 4.1]
+ - bnd-Part: [e.g. bnd-lib, bnd-cli, gradle, maven, ant, libg]
+ - OS: [e.g. Linux, Windows, iOS]
+ - Java-Version: [e.g. openjdk-11]
+ - OSGi-Framework: [e.g. Felix, Equinox]
+ - Related Spec: [e.g. 112.3.4  Declarative Services - Constructor Injection ]
```

Comment:
You should also mention the `Git-*` headers. See https://github.com/bndtools/bnd/blob/master/CONTRIBUTING.md#reporting-issues

Reply:
done!